sklearn/feature_extraction/text.py

```diff
@@ -615,6 +615,13 @@ class CountVectorizer(BaseEstimator, VectorizerMixin):
     See also
     --------
     HashingVectorizer, TfidfVectorizer
+
+    Notes
+    -----
+    The ``stop_words_`` attribute can get large which may not be desirable when
```

Comment:
I would document it at the attribute documentation, saying "only for introspection and can savely be removed before pickling" or something like that.